src/utils/daos.c

```diff
@@ -1212,9 +1214,14 @@ cont_op_hdlr(struct cmd_args_s *ap)
 		uuid_generate(ap->c_uuid);
 
 	if (op != CONT_CREATE && op != CONT_DESTROY) {
-		rc = daos_cont_open(ap->pool, ap->c_uuid,
-				    DAOS_COO_RW | DAOS_COO_FORCE,
-				    &ap->cont, &cont_info, NULL);
+		if (ap->cont_label)
+			rc = daos_cont_open_by_label(ap->pool, ap->cont_label,
+					     DAOS_COO_RW | DAOS_COO_FORCE,
+					     &ap->cont, &cont_info, NULL);
```

Comment:
As above.  You already have cont_info here so copying the uuid to c_uuid would be a good idea.